enterprise_subsidy/apps/fulfillment/api.py

```diff
@@ -32,3 +37,103 @@ def determine_fulfillment_client(subsidy_uuid, content_key):
     return None
     """
     raise NotImplementedError
+
+
+class FulfillmentException(Exception):
+    pass
+
+
+class InvalidFulfillmentMetadataException(FulfillmentException):
+    pass
+
+
+class GEAGFulfillmentHandler():
+    """
+    A class for fulfilling a GEAG transaction.
+    """
+    CENTS_PER_DOLLAR = 100.0
```

Comment:
nit/curious: Should we be reusing the existing `CENTS_PER_DOLLAR` constant defined [here](https://github.com/openedx/enterprise-subsidy/blob/30028f60e92bfce584d84513eaf16704299dc149/enterprise_subsidy/apps/subsidy/constants.py#L6)?